### DISCUSSION TITLE
230918 스프린트 회의 (14:00 ~ )

### DISCUSSION THREAD

**Discussion by somsom13** [woowacourse-teams/2023-shook#419] [회의]
## 회의 안건
9월 18일~ 일주일 진행되는 스프린트를 정한다.

**Comment by Cyma-s** [comment]
## 회위회 - 스프린트 목적을 정하는 회의

- 예상 목적
    - 스프린트 목적을 정한다.
- 예상 시간
    - 30분

## 스프린트 목적을 정하는 회의

2023년 9월 18일 오후 2:03

### 스프린트 목적 제안

- **5차 데모데이 요구사항 충족**
- 데모데이 때 발표할 기능 개발
    - 노래 카테고리 (사이트별, 장르별, 가수별)
        - 추가 데이터 수집이 필요할 듯
    - 검색 활성화 후, 메인에 보여지는 차트 노래를 줄이는 방향. 존재하는 노래만 보여준 뒤, 키워드로 검색하는 필터링 느낌. (쌩 검색 X)
    - 킬링파트 추가 요청 페이지 ⇒ 내가 더 마음에 드는 킬링파트를 요청할 수 있는 등록 버튼을 만들어준다.
    - 킬링파트 말고 노래 추가 버튼
- 요청을 보낼 수 있는 창구
    - 채널톡

### FE

- 기능은 개발할 수 있는 상황
- 예외 처리

### BE

- 아무것도 안 됐음
    - 인덱스: 쿼리 정리 완료. 인덱스 설정 아직 안 함. 설정할 수 없는 쿼리의 이유 조사.
    - 톰캣: 테스트 몇 개는 있음. 설정 안 함.
- ako: 기능 개발은 어려울 것 같다.
- split: 분업하자.
- baron: 기능이면 분업 하고 싶네요.
- vero: 내일이나 모레 아침~ 낮까지 요구사항이 완료되면 기능 개발해도 된다.

2023년 9월 18일 오후 2:35

## 회위회 - 스프린트 회의

- 예상 목적
    - 개발할 기능이 생긴다.
        - 카카오 로그인 할 지 결정한다.
        - jwt 리프레쉬 토큰 어떻게 처리할 지 결정한다.
    - 스프린트 일정 정하기
    - 발표자 정하기
- 예상 시간
    - 20분

## 스프린트 회의

### 기능 개발 인원

- FE: 코난, 도밥
- BE: 아코, 베로, 바론

### 개발할 기능

- **카카오 로그인 (몹 프로그래밍) + jwt 리프레시 토큰 처리 (3)**
    - BE 아코
    - FE 우코
- **장르 카테고리 추가 (2)**
- **스와이프 가이드 시트 (1)**
- 아코, 바론: 가수 필터링, 장르 카테고리 구현하기에 기간 상 적합해보인다. 1니즈 2제안
- 베로, 코난, 스플릿: 킬링파트 추가 요청 페이지 ⇒ 내가 더 마음에 드는 킬링파트를 요청할 수 있는 등록 버튼 만들기 3.5니즈
- 베로, 코난, 도밥: 노래 추가 페이지 만들기 3니즈
    - 노래 들을 게 없어서 노래 추가 페이지가 필요했을 수도 있다.
    - 그렇지만 사실상 모든 데이터를 넣을 수는 없기 때문에 어쨌든 반대도 동일하다.
- 베로, 우코: 플레이리스트 같은 큐레이션 만들기 1제안
- 데스크탑에서 폭이 줄어들게 변경

### 장르 카테고리

- 메인 페이지에서 보여질 모습
    - 메인에서 카테고리 버튼을 누르면 다른 페이지로 이동 (다른 페이지 이동형)
    - 메인에서 카테고리 버튼을 누르면 메인 페이지의 노래가 변경
    - 메인에서 카테고리를 여러 개 보여주기 (메인페이지 분리형) **⇒ 1번**
    - 메인에서 카테고리 여러 개 보여주고, 더보기 누르면 다른 페이지로 이동한다. 메인에서 보여지는 카테고리의 노래는 모든 노래가 아닌, 제한적인 수의 노래를 보여준다. ⇒ **2번**

### 일정

- 기능 개발 마감일
    - 로그인 ⇒ 오늘 최대한 마무리, 수요일 밤까지 진행
    - BE 카테고리 ⇒ 화요일 밤까지
    - BE 데이터 수집 및 테이블 변경 + 데이터 수정 ⇒ 수요일 밤까지
    - BE 성능 개선 ⇒ 화요일 밤까지
    - FE 카테고리 ⇒ 수요일 밤까지
    - FE 스와이프 가이드 ⇒ 수요일 밤까지
    - FE 브라우저 테스트 문서화 ⇒ 수요일 밤까지
- 발표자
    - 베로
    - 코난 or 도밥

회의 마무리: 2023년 9월 18일 오후 5:24
